docs/intro/install.rst

```diff
@@ -69,20 +69,25 @@ Overview
 Upload Elgg
 -----------
 
--  Download the `latest version of Elgg`_
--  Upload the ZIP file with an FTP client to your server
--  Unzip the files in your domain's document root (/home/username/www).
-
-.. _latest version of Elgg: http://elgg.org/download.php
-
-Elgg also supports installing from source with Composer:
+With Composer (recommended):
 
 .. code:: shell
 
     cd /path/to/wwwroot/
     composer self-update
     composer global require "fxp/composer-asset-plugin:~1.0"
-    composer create-project Elgg/Elgg:~2.0 .
+    composer create-project elgg/starter-project:dev-master .
```

Comment:
That's a good question. Maybe starter-project should specify the version of elgg. Will we even want/need to put versions on starter project?